packages/core/src/browser/style/index.css

```diff
@@ -99,12 +99,30 @@ blockquote {
   bottom: 0;
   z-index: 50000;
   background: var(--theia-editor-background);
-  background-image: var(--theia-preloader);
   background-size: 60px 60px;
   background-repeat: no-repeat;
   background-attachment: fixed;
   background-position: center;
   transition: opacity 0.8s;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.theia-preload::after {
+  animation: 1s theia-preload-rotate infinite;
+  color: #777; /* color works on both light and dark themes */
```

Comment:
Unfortunately we cannot re-use a variable here like `var(theia-foreground` since it is not ready and would cause a flicker. Instead we can use a color which should work for most themes.